bin/cli.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const readline = require('readline');
+
+const Cli = require('../lib/cli/cli');
+
+let rl = null;
+
+const log = (msg) => {
+  const userInput = rl.line;
+  if (userInput) rl.clearLine();
+  console.log(msg);
```

Comment:
And it should write to the underlying stream directly (`process.stdout` in this case, but it may be anything that the readline interface was created on top of) instead of using `console.log()`.